### ISSUE TITLE
some GRASS plugins doesn't work in QGIS 3.4 (missing description txt files)

### ISSUE THREAD

**Issue by rsmaior** [qgis/QGIS#32466]
I need to use i.rectify GRASS algorithm from QGIS 3.4.5 in Linux. But I noted it was not there. Then I tested in Windows and others Linux computers. I had the same result. 

Then I tested in QGIS 2.18, and worked (both Linux and Windows).

Looking here https://github.com/qgis/QGIS/tree/release-2_18/python/plugins/processing/algs/grass7/description I can see there is i.rectify.txt and many others

But looking https://github.com/qgis/QGIS/tree/release-3_4/python/plugins/processing/algs/grass7/description I can see there's no i.rectify.txt and there's others missing 

Likewise https://github.com/qgis/QGIS/tree/release-3_6/python/plugins/processing/algs/grass7/description, https://github.com/qgis/QGIS/tree/release-3_8/python/plugins/processing/algs/grass7/description and https://github.com/qgis/QGIS/tree/release-3_10/python/plugins/processing/algs/grass7/description I can confirm QGIS 3.6, 3.8 and 3.10 doesn't have these files too.

But looking https://github.com/qgis/QGIS/tree/release-3_2/python/plugins/processing/algs/grass7/description and https://github.com/qgis/QGIS/tree/release-3_0/python/plugins/processing/algs/grass7/description the files are there

**How to Reproduce**

1) With QGIS 3.4 open processing Toolbox search and type i.rectify and no results will appear
2) With QGIS 2.18 open processing Toolbox search and type i.rectify and the expected algorithm will appear

**Additional context**

I did not tested this with QGIS 3.0, 3.2, 3.6, 3.8 or 3.10, but I strongly suspect that 3.0 and 3.2 will work (since they have the files) and that 3.6, 3.8 and 3.10 won't (because they don't have the files).

**Comment by gioman** [comment]
> I did not tested this with QGIS 3.0, 3.2, 3.6, 3.8 or 3.10, but I strongly suspect that 3.0 and 3.2 will work (since they have the files) and that 3.6, 3.8 and 3.10 won't (because they don't have the files).

nope, is missing in 3.* period. I think you have seen "v.rectify", that is another tool.



> I need to use i.rectify

can you spot other missing tools in Processing (compared to 2.18)?

**Comment by rsmaior** [comment]
**Avaiable in QGIS 2.18 but not in QGIS 3.4**
i.eb.hsebal01
i.rectify
r.aspect
r.contour.level
r.contour.step
r.cost.cost
r.cost.points
r.cost.raster
r.flow.aspect
r.flow.aspect.barrier
r.flow.barrier
r.lake.coords
r.lake.layer
r.mapcalc
r.median
r.mfilter.fp
r.random.raster
r.reclass.area.greater
r.reclass.area.lesser
r.slope
r.support
v.buffer.column
v.buffer.distance
v.clean.advanced
v.generalize.displacement
v.generalize.network
v.generalize.simplify
v.generalize.smooth
v.in.ascii.points
v.in.ascii.standard
v.lrs.create
v.lrs.segment.where
v.net.arcs
v.net.articulation
v.net.connect
v.net.nodes
v.split.length
v.split.vert
v.surf.bspline.lambda
v.surf.bspline.sparse
v.surf.rst.line
v.to.rast.attribute
v.to.rast.value
v.voronoi.skeleton
v.what.rast.centroids
v.what.rast.points

**Avaiable in QGIS 3.4 but not in QGIS 2.18**
r.contour
r.cost
r.fill.stats
r.lake
r.mapcalc.simple
r.out.ascii
r.out.mat
r.out.mpeg
r.out.png
r.out.pov
r.out.ppm3
r.out.vtk
r.out.xyz
r.proj
r.reclass.area
r.stats.zonal
v.buffer
v.cluster
v.decimate
v.distance
v.generalize
v.in.ascii
v.in.e00
v.net
v.net.timetable
v.out.ascii
v.out.postgis
v.out.svg
v.proj
v.split
v.to.rast
v.what.rast

**Comment by rsmaior** [comment]
I listed both missing QGIS 2.18 → 3.4 and 3.4 → 2.18 because I think it's possible that some routines that are missing from one simply changed the name or was incorporated by a more generic routine. 

i.e. 2.18 has v.what.rast.centroids and v.what.rast.points routines, but 3.4 has v.what.rast which englobes the other two by adding an internal option to select points or centroids.

But I could be wrong for that routine and others, so I thought it would be safer to include anything

**Comment by Pedro-Murteira** [comment]
Still valid on QGIS 3.22.3.

**Comment by alexbruy** [comment]
IMHO this should be feature, not bug.
